src/styles/blockmain.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  width: $width-set-card;
+  padding: 50px $side-padding;
+  margin: auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/211388944-8d6c0fc2-4190-4c70-991a-91a0460575ed.png)